SWLOR.Game.Server/Feature/LootTableDefinition/DantooineLootTableDefinition.cs

```diff
@@ -0,0 +1,507 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Service.LootService;
+
+namespace SWLOR.Game.Server.Feature.LootTableDefinition
+{
+    public class DantooineLootTableDefinition: ILootTableDefinition
+    {
+        private readonly LootTableBuilder _builder = new();
+
+        public Dictionary<string, LootTable> BuildLootTables()
+        {
+            QueenKinrath();
```

Comment:
From a player perspective, I would be very disappointed to learn that a brand new planet doesn't have any unique or new loot. From what I saw, everything here was a copy/paste from other planets. I would strongly suggest creating some unique items to make Dantooine more interesting to visit. As it sits, it feels a lot like a clone of existing content which I personally don't find too interesting.

Reach out in Discord if you want to discuss ideas.